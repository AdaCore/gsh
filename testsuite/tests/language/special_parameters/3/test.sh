$SHELL ./subtest.sh
$SHELL ./subtest.sh ""
$SHELL ./subtest.sh "a b" "c"

