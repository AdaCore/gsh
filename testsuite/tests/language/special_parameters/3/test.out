

Quoted @
NonQuoted @


Quoted @
param: ''
NonQuoted @
a b c
a b c
Quoted @
param: 'a b'
param: 'c'
NonQuoted @
param: 'a'
param: 'b'
param: 'c'
